python/test/test_vectors.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
```

Comment:
This file is not executable, might as well remove the shebang.

Reply:
Fixed 👍